dask_jobqueue/job.py

```diff
@@ -0,0 +1,394 @@
+from contextlib import contextmanager
+import logging
+import os
+import re
+import shlex
+import subprocess
+import six
+import sys
+import weakref
+
+import dask
+from distributed.deploy.spec import ProcessInterface, SpecCluster
+from distributed.scheduler import Scheduler
+
+from distributed.utils import format_bytes, parse_bytes, tmpfile, get_ip_interface
+
+logger = logging.getLogger(__name__)
+
+
+class Job(ProcessInterface):
+    """ Base class to launch Dask workers on Job queues
```

Comment:
We should change the docstring here and below, shouldn't we? I guess this is already planned.